contracts/src/deployers/reth/RETHHyperdriveDeployerCoordinator.sol

```diff
@@ -21,11 +20,9 @@ contract RETHHyperdriveDeployerCoordinator is HyperdriveDeployerCoordinator {
     using SafeERC20 for ERC20;
     using FixedPointMath for uint256;
 
-    /// @notice The Rocket Storage contract.
-    IRocketStorage public immutable rocketStorage;
```

Comment:
I think removing this is a good call, but we'll want to feed in `rocketTokenReth` through the `RETHHyperdriveDeployerCoordinator`.